libgsf.yaml

```diff
@@ -40,16 +40,9 @@ pipeline:
   - uses: git-checkout
     with:
       repository: https://gitlab.gnome.org/GNOME/libgsf.git
-      expected-commit: ea9d8cd1369661bf62d0476474700a9e0887c812
+      expected-commit: 64a465dd1a12f3590b6cc30fe7a97c841794cf9a
       tag: LIBGSF_${{vars.mangled-package-version}}
 
-  - runs: |
-      git fetch origin master:master
-      # Fix failure to build from source against new libxml2, without
-      # nanohttp support.
-      git show 5d4bb55095d3d6ef793c1908a88504183e28644c -- configure.ac gsf/gsf-input-http.c > partial-cherry-pick.patch
-      git apply partial-cherry-pick.patch
-
```

Comment:
this commit was part of the new release so we can remove this. (ref: https://gitlab.gnome.org/GNOME/libgsf/-/commits/LIBGSF_1_14_53/)